plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/dialogs/MultiPageWizardDialog.java

```diff
@@ -159,6 +171,34 @@ protected Control createContents(Composite parent) {
         return contents;
     }
 
+    private void preLoadPages() {
+        pageArea.setRedraw(false);
+        try {
+            for (IWizardPage page : wizard.getPages()) {
+                page.createControl(pageArea);
+                Control pageControl = page.getControl();
+                applyDialogFont(pageControl);
+                if (page instanceof ActiveWizardPage activeWizardPage) {
+                    activeWizardPage.updatePageCompletion();
+                    activeWizardPage.activatePage();
+                }
```

Comment:
Why do you hide page in such a weird way? Is there any API for page activation? I thought we already use it.

Reply:
We need to create controls. We can't activate pages without them because many pages use their UI components during activation.
By doing it that way and by forbidding any draw operations, there will be no notable visual changes during page load.